apps/lfx-one/src/app/modules/dashboards/dashboard.component.html

```
@@ -1,17 +1,18 @@
 <!-- Copyright The Linux Foundation and each contributor to LFX. -->
 <!-- SPDX-License-Identifier: MIT -->
 
-@switch (dashboardType()) {
-  @case ('contributor') {
-    <lfx-contributor-dashboard />
+@switch (activeLens()) {
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)